docs/data-guidelines.md

```diff
@@ -56,13 +56,16 @@ For example, the feature for a `focus` event targeting the `Element` interface w
 }
 ```
 
-This rule applies to the event features themselves, not the features for the event handlers. For example, `focus_event` and `onfocus` are two separate features.
+The event handler `onfocus` is included in the `focus_event` entry. If an implementation doesn't support the event handler property, use `partial_implementation` with the note `"The <code>onfocus</code> event handler property is not supported."`. If an implementation doesn't support the event name, use `partial_implementation` with the note `"The <code>focus</code> event name is not supported."`.
```

Comment:
Two suggestions here, for clarity:

- I think it's worth including an explicit prohibition on creating `on` entries (i.e., having an imperative like "Don't create features for `on` event handler properties"). Perhaps saying "represented by" instead of "included in" would also help? As written, I think there's a plausible interpretation here that `on` handlers can be subfeatures of the event feature.
- "If an implementation doesn't support the event name…" is slightly ambiguous. It could be read as if all events are either fully supported or partially supported and none are false. Perhaps something like "If only the handler is supported…"

Reply:
For the "If an implementation doesn't support the event name" case, I think we should treat that as not supported, meaning no `"version_added": false`. It is possible that on* properties exist and will mislead feature detection, but I don't think the value of trying to capture this correctly justifies the work.

Although I will say that one benefit of representing it is that if tests using the on* properties show earlier support than BCD, then such `partial_implementation` entries would avoid incorrectly updating BCD to claim earlier support.

Reply:
I think I addressed Daniels suggestions.

And I agree with Philip to make it `"version_added": false` when on* event handler exists but no event name.